### PR TITLE
feat: Payment pointer unit test

### LINKED ISSUE #684: Payment Pointer Controller tests
Payment Pointer controller tests should be created

### PR DESCRIPTION
**Pull request by rico191013**
### Context

- fixes #684

### CODE REVIEW

File: packages/wallet/backend/tests/paymentPointer/controller.test.ts [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+    createMockPaymentPointerControllerDeps()
+  })
+
+  beforeEach(async (): Promise<void> => {
+    await createUser({ ...args, isEmailVerified: true })
+    await createReqRes()
+  })
+
+  afterAll(async (): Promise<void> => {
+    appContainer.stop()
```
Comment (dragosp1011) [inline packages/wallet/backend/tests/paymentPointer/controller.test.ts:125]: add `await` for methods in`afterAll`

File: packages/wallet/backend/tests/paymentPointer/controller.test.ts [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+      req.params = {
+        accountId: account.id
+      }
+      await ppController.listAll(req, res, next)
+      expect(res.statusCode).toBe(200)
+      expect(res._getJSONData()).toMatchObject({
+        success: true,
+        message: 'SUCCESS'
+      })
+      expect(res._getJSONData()).toHaveProperty('data')
```
Comment (dragosp1011) [inline packages/wallet/backend/tests/paymentPointer/controller.test.ts:210]: can `data` be checked as well in all tests?

**Review by raducristianpopa — COMMENTED** [review]
I think these tests needs to be extended now, since we added a new type of payment pointers (for web monetization).

**Review by dragosp1011 — COMMENTED** [review]
rename the directory as well

**Review by dragosp1011 — APPROVED** [review]

**Review by dragosp1011 — APPROVED** [review]

**Pull request merged**